fly.toml

```diff
@@ -0,0 +1,43 @@
+# fly.toml file generated for fcc-seoul on 2022-11-28T00:17:04+09:00
```

Comment:
Will this file ever require changes or will contributors need to worry about this file when making local changes? I haven't used `fly.dev` before so I'm not sure if it complicates the process at all.

Reply:
This is related to deployment, so they won't need any changes, at least in the near future.
People won't need to worry about this file or making changes here.